entries/open-graph/SearchPreview.tsx

```diff
@@ -0,0 +1,70 @@
+import { date as wpDate } from '@wordpress/date';
+import { useMedia } from '@alleyinteractive/block-editor-tools';
+
+interface SearchPreviewProps {
+  title: string;
+  description: string;
+  imageId: number;
+  date: string;
+  link: string;
+  postType: string;
+  siteTitle: string;
+  siteIcon: string;
```

Comment:
```suggestion
  siteIcon?: string;
```
and is this a string?